sdk/scripts/Build-OpenCV.ps1

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env pwsh
+
+Param(
+    [Parameter(Position = 0)]
+    [ArgumentCompleter({
+        $PossibleValues = @('macOS', 'Mac-Catalyst', 'iOS', 'iOS-Simulator', 'Web')
+        return $PossibleValues | ForEach-Object { $_ }
+    })]
+    [String]$Platform,
+
+    [Parameter(Position = 1)]
+    [ArgumentCompleter({
+        $PossibleValues = @('AMD64', 'ARM64', 'WASM32')
+        return $PossibleValues | ForEach-Object { $_ }
+    })]
+    [String]$Architecture
+)
+
+If(-Not $Platform) {
+    Write-Error -Message "Parameter '-Platform' is not specified."
+    Exit 1
+}
+If(-Not $Architecture) {
+    Write-Error -Message "Parameter '-Architecture' is not specified."
+    Exit 1
+}
+
+Push-Location -Path "sdk/"
+Write-Host "Cloning OpenCV..."
+
+# Ensure the "third-party" directory exists
+if (-Not (Test-Path "third-party")) {
+    New-Item -ItemType Directory -Path "third-party" | Out-Null
+}
+Push-Location -Path "third-party/"
+
+# Clone the OpenCV repository
+git clone git@github.com:opencv/opencv.git
```

Comment:
this is probably ok for now, but in future we may want to lock in to a specific version and use a submodule